.github/workflows/release.yml

```diff
@@ -48,11 +48,11 @@ jobs:
         include:
         - name: ubuntu-20.04
           os: ubuntu-20.04
-          compiler: clang++-9
+          compiler: clang++-10
           artifact_path: /build/src/Grazer/grazer
           asset_name: grazer
           asset_description: Binary for ubuntu-20.04
-          dep-install: sudo apt-get install -y cmake doxygen graphviz
+          dep-install: sudo apt-get install -y cmake clang-10 graphviz
```

Comment:
Why is doxygen not listed anymore?

Reply:
because this is the matrix build to generate the binary artifacts (these only include grazer itself and thus do not require doxygen)